queries.sql

```diff
@@ -131,4 +131,87 @@ ORDER BY num_animals DESC
 LIMIT 1;
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221071927-cd581c4b-121d-47d5-b7c1-4ee0b1e8c97f.png)


**[Optional]**

- [ ] It would be nice if you make all your commit messages capitalize to make your pull request descriptive and professional :+1:

Reply:
I have taken notice of this suggestion. I will adhere to it. 

Thank you for the feedback